src/SprykerEco/Zed/AmazonPay/Business/Payment/Handler/Ipn/IpnAbstractTransferRequestHandler.php

```diff
@@ -80,7 +79,7 @@ public function handle(AmazonpayIpnPaymentRequestTransfer $paymentRequestTransfe
                 });
 
                 break;
-            } catch (Exception $e) {
+            } catch (\Throwable $e) {
```

Comment:
That is usually not advised for normal exception handling. Lower than exception is an internal error that shouldnt be caught IMO. So the old typehint can and probably should be staying.